### PR TITLE
CORE-18548: Add link manager boot configuration path

### PR DESCRIPTION
**Pull request by yift-r3**
See runtime pull request in https://github.com/corda/corda-runtime-os/pull/5235

### CODE REVIEW

**Review by williamvigorr3 — DISMISSED** [review]
LGTM

File: gradle.properties [hunk 1]
```diff
@@ -5,7 +5,7 @@ cordaProductVersion = 5.2.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 12
```
Comment (vkolomeyko) [inline gradle.properties]: Are we sure this is a breaking change?

**Review by vkolomeyko — DISMISSED** [review]
LGTM

**Review by williamvigorr3 — APPROVED** [review]
LGTM

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Pull request merged**